runner/docker/Dockerfile.live-base-comfyui

```diff
@@ -21,6 +21,27 @@ RUN cd /comfyui/custom_nodes && \
     cd ComfyUI-Depth-Anything-Tensorrt && \
     pip install -r requirements.txt
 
+# Install ComfyUI-StreamDiffusion (https://github.com/pschroedl/ComfyUI-StreamDiffusion.git)
+RUN git clone https://github.com/pschroedl/ComfyUI-StreamDiffusion.git && \
+    cd ComfyUI-StreamDiffusion && \
+    pip install -r requirements.txt && \
+    cd ..
+
+# Install ComfyUI-SAM2-Realtime (https://github.com/pschroedl/ComfyUI-SAM2-Realtime.git)
+RUN git clone -b relax_python_3_10 https://github.com/pschroedl/ComfyUI-SAM2-Realtime.git && \
+    cd ComfyUI-SAM2-Realtime && \
+    pip install -r requirements.txt && \
+    cd ..
```

Comment:
FTR the `RUN` commands from docker instantiate their own shell. Changing directory will not change the directory for the next docker commands. So no need to restore the previous workdir!

Reply:
Good heads up - removed those and made sure we were cloning each into the correct directory ( could use WORKDIR I suppose

Reply:
Yeah! I think a `WORKDIR` on the `custo_nodes` folder and then setting up all the custom nodes in a contiguous "section" would look good :). Then also don't even need the `cd` before each clone